src/bluesearch/mining/eval.py

```diff
@@ -523,20 +596,28 @@ def ner_errors(
         return "\n".join(out)
 
 
-def ner_confusion_matrix(iob_true, iob_pred, normalize=None, mode="entity"):
+def ner_confusion_matrix(
+    iob_true: pd.Series,
+    iob_pred: pd.Series,
+    normalize: Optional[str] = None,
+    mode: Optional[str] = "entity",
```

Comment:
Same as above

Reply:
Fixed all wrong usages of Optiona[...] in 1ecd77b.